code-of-conduct-languages/ar.md

```diff
@@ -0,0 +1,45 @@
+<div dir="rtl">
+
+##  مدونة قواعد السلوك لمجتمع CNFC النسخة v1.0
+
+### قواعد السلوك للمساهمين
+
+كمساهمين و مشرفين على هذا المشروع، و في مصلحة تعزيز
+مجتمع مفتوح و مرحّب به ، نتعهد باحترام جميع الأشخاص الذين يساهمون
+من خلال الإبلاغ عن المشكلات و نشر طلبات الميزات و تحديث الوثائق
+و تقديم طلبات السحب أو التصحيحات و غيرها من الأنشطة.
+
+نحن ملتزمون بجعل المشاركة في هذا المشروع تجربة خالية من المضايقات للجميع.
+بغض النظر عن مستوى الخبرة، أو الجنس، أو الهوية الجنسية و التعبير عنها، 
+أو التوجه الجنسي، أو الإعاقة، أو المظهر الشخصي، أو حجم الجسد، أو العرق، أو اﻷصل العرقي، أو العمر،
+أو الدين، أو الجنسية.
+
+من الأمثلة على السلوك غير المقبول للمشاركين ما يلي:
+
+* استخدام الصور أو اللغة الجنسية 
+* الهجمات الشخصية
+* تعليقات مهينة أو التصيد
+* مضايقة عامة أو خاصة
+* نشر المعلومات الخاصة للآخرين، مثل العناوين الحقيقية أو البريد  الإلكتروني دون إذن صريح
+* سلوك غير أخلاقي أو غير مهني
+
+يكون لمشرفي المشروع الحق و المسؤولية في إزالة أو تعديل أو رفض
+التعليقات و التعهدات و التضمين و تعديلات المشكلات و المساهمات الأخرى التي لا تتوافق مع 
+مدونة السلوك هذه. و باعتماد مدونة قواعد السلوك هذه، يلتزم مشرفو المشروع
+بتطبيق هذه المبادئ تطبيقا عادلا و متسقا على جميع جوانب.
+قد يتم حذف مشرفي المشروع الذين لا يتبعون أو لا يطبقون المدونة نهائيًا من طرف فريق المشروع.
+
+تنطبق قواعد السلوك هذه داخل مساحات المشروع و في الأماكن العامة
+عندما يمثل الفرد المشروع أو مجتمعه.
+
+يمكن الإبلاغ عن حالات السلوك التعسفي أو المضايقة أو السلوك الغير المقبول في Kubernetes من خلال الاتصال بـ [لجنة قواعد سلوك Kubernetes](https://git.k8s.io/community/committee-code-of-conduct) عبر <conduct@kubernetes.io>. بالنسبة للمشاريع الأخرى ، يرجى الاتصال بمشرف مشروع CNCF أو وسيطنا ، Mishi Choudhary <mishi@linux.com>.
```

Comment:
![MD](https://user-images.githubusercontent.com/15904286/69893510-8f230f00-1312-11ea-85a3-99d1e031f8f5.png)
Does not look like a well formatted paragraph

Reply:
Github does not support right to left in PR diff but if you visit the page from the repo it renders correctly:
https://github.com/10ten-me/foundation/blob/master/code-of-conduct-languages/ar.md
![image](https://user-images.githubusercontent.com/42375159/70899844-fc23ed80-2010-11ea-9cd8-8ab6f7167c04.png)